crypto/ledger/ledger_secp256k1.go

```diff
@@ -210,11 +212,19 @@ func warnIfErrors(f func() error) {
 }
 
 func convertDERtoBER(signatureDER []byte) ([]byte, error) {
-	sigDER, err := btcec.ParseDERSignature(signatureDER, btcec.S256())
+	sigDER, err := ecdsa.ParseDERSignature(signatureDER)
 	if err != nil {
 		return nil, err
 	}
-	sigBER := tmbtcec.Signature{R: sigDER.R, S: sigDER.S}
+
+	sigStr := sigDER.Serialize()
+	var r, s big.Int
+	// The format of a DER encoded signature is as follows:
+	// 0x30 <total length> 0x02 <length of R> <R> 0x02 <length of S> <S>
```

Comment:
Might seem paranoid, but I suggest that we add a length minimum check here
```go
// Validating that we have a somewhat proper DER signature.
if n := len(sigDER); n < 4 {
    return nil, fmt.Errorf("invalid DER length: want >=4 got %d", n)
}
if g, w := sigDER[0], 0x30; g != w {
    return nil, fmt.Errorf("invalid DER: byte[0]=%x, want=%x", g, w)
}
if wantLen, gotLen := int(sigDER[1]), len(sigDER[3:]); gotLen != wantLen {
    return nil, fmt.Errorf("invalid DER len: got=%d, want=%d",
       gotLen, wantLen)
}

// Eliminate compiler Bounds checking.
_ = sigDER[9:]
```

Reply:
Seems a bit paranoid indeed, given that `Serialize` always returns this format.
https://go.dev/play/p/WuZKekW5s9x

Reply:
Cool no biggie.